api/impl/retrieval_client.go

```diff
@@ -5,20 +5,31 @@ import (
 	"io"
 
 	"gx/ipfs/QmR8BauakNcBa3RbE4nbQu76PDiJgoQgz8AJdhJuiU4TAw/go-cid"
+	"gx/ipfs/QmTu65MVbemtUxJEWgsTtzv9Zv9P8rvmqNA4eG9TrTRGYc/go-libp2p-peer"
 
 	"github.com/filecoin-project/go-filecoin/address"
+	"github.com/filecoin-project/go-filecoin/types"
 )
 
 type nodeRetrievalClient struct {
-	api *nodeAPI
+	api      *nodeAPI
+	plumbing retrievalClientPorcelainAPI
```

Comment:
I'm confused; it's a PorcelainAPI but it's called plumbing?

Reply:
Sorry, this is just bad naming on my part